systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -1066,6 +1084,301 @@ void testRestoreOffsetsInConsumerGroup(ExtensionContext extensionContext) {
+        final String kafkaClientsName = mapWithKafkaClientNames.get(extensionContext.getDisplayName());
+        String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
+        String kafkaClusterSourceName = clusterName + "-source";
+        String kafkaClusterTargetName = clusterName + "-target";
+        String topicSourceName = MIRRORMAKER2_TOPIC_NAME + "-" + rng.nextInt(Integer.MAX_VALUE);
+        String topicTargetName = kafkaClusterSourceName + "." + topicSourceName;
+        String kafkaUserSource = clusterName + "-my-user-source";
+        String kafkaUserTarget = clusterName + "-my-user-target";
+        String kafkaUserTlsSource = clusterName + "-tls-user-source";
+        String kafkaUserTlsTarget = clusterName + "-tls-user-target";
+        String kafkaTlsListener = "mytls";
+
+        // Deploy source kafka with tls listener and SCRAM-SHA authentication
+        resourceManager.createResource(extensionContext, KafkaTemplates.kafkaEphemeral(kafkaClusterSourceName, 3, 3)
+            .editSpec()
+            .editKafka()
+            .withListeners(
+                new GenericKafkaListenerBuilder()
+                    .withName(Constants.TLS_LISTENER_DEFAULT_NAME)
+                    .withPort(9093)
+                    .withType(KafkaListenerType.INTERNAL)
+                    .withTls(true)
+                    .withAuth(new KafkaListenerAuthenticationScramSha512())
+                    .build(),
+                new GenericKafkaListenerBuilder()
+                    .withName(kafkaTlsListener)
+                    .withPort(9094)
+                    .withType(KafkaListenerType.INTERNAL)
+                    .withTls(true)
+                    .withAuth(new KafkaListenerAuthenticationTls())
+                    .build())
+            .endKafka()
+            .endSpec()
+            .build());
+
+        // Deploy target kafka with tls listener and SCRAM-SHA authentication
+        resourceManager.createResource(extensionContext, KafkaTemplates.kafkaEphemeral(kafkaClusterTargetName, 3, 3)
+            .editSpec()
+            .editKafka()
+            .withListeners(
+                new GenericKafkaListenerBuilder()
+                    .withName(Constants.TLS_LISTENER_DEFAULT_NAME)
+                    .withPort(9093)
+                    .withType(KafkaListenerType.INTERNAL)
+                    .withTls(true)
+                    .withAuth(new KafkaListenerAuthenticationScramSha512())
+                    .build(),
+                new GenericKafkaListenerBuilder()
+                    .withName(kafkaTlsListener)
+                    .withPort(9094)
+                    .withType(KafkaListenerType.INTERNAL)
+                    .withTls(true)
+                    .withAuth(new KafkaListenerAuthenticationTls())
+                    .build())
+            .endKafka()
+            .endSpec()
+            .build());
```

Comment:
Same as above.
Also the comment one line above isn't right FMPOV - you are deploying Kafka with TLS listener & TLS auth and TLS listener & SCRAM-SHA auth